docs/source/_templates/autosummary/base.rst

```diff
@@ -1,7 +1,7 @@
 {% if "modin.pandas" in fullname %}
     {% set fullname = "modin.pandas." + objname %}
 {% endif %}
-{% if "modin.pandas.general" in module %}
+{% if "modin.pandas.general" in module or ("modin.pandas" in module and name in ["read_csv", "read_json", "read_parquet", "read_snowflake", "to_snowpark", "to_pandas"])%}
```

Comment:
can we just do `if "modin.pandas.io" ...`

Reply:
No, unfortunately that does not work - the input/output functions are not in `modin.pandas.io` module, just `modin.pandas`. Replacing with `modin.pandas` raises a lot of warnings (100+) so it's probably better to have this bad code here haha.

Reply:
this will be hard to maintain. Can you add a jira to track this?

Reply:
Yes - here's the JIRA https://snowflakecomputing.atlassian.net/browse/SNOW-1439522